source/renderer/app/components/wallet/settings/WalletSettingsRemoveConfirmationDialog.tsx

```diff
@@ -68,10 +67,7 @@ const WalletSettingsRemoveConfirmationDialog = observer((props: Props) => {
       !isWalletNameConfirmationCorrect);
   // @ts-ignore ts-migrate(2554) FIXME: Expected 2 arguments, but got 1.
   const handleSubmit = React.useCallback(() => !isDisabled && onContinue());
-  const buttonClasses = classnames([
-    'attention',
-    isSubmitting ? styles.isSubmitting : null,
```

Comment:
is that really safe to remove?

Reply:
Same for `isSubmitting` on `source/renderer/app/components/wallet/settings/DeleteWalletConfirmationDialog.scss`